cpp/dolfinx/mesh/graphbuild.cpp

```diff
@@ -38,8 +38,10 @@ namespace
 /// cell.
 /// @param[in] local_graph The dual graph for cells on this MPI rank
 /// @return (0) Extended dual graph to include ghost edges (edges to
-/// off-rank cells) and (1) the number of ghost edges
-std::pair<graph::AdjacencyList<std::int64_t>, std::int32_t>
+/// off-rank cells), (1) the number of ghost edges, (2) the vertices lying on
```

Comment:
Could you clarify the jargon? The docstring has 'edges' which I presume are edges in the graph sense (and not mesh sense), and also 'nodes' which seems to be in the mesh sense (and not graph sense).